ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/Constants.java

```diff
@@ -184,8 +185,8 @@ public String toString() {
         httpCodeMap.put("LoopDetected", "508");
         httpCodeMap.put("NotExtended", "510");
         httpCodeMap.put("NetworkAuthenticationRequired", "511");
-        httpCodeMap.put("NetworkAuthorizationRequired", "511"); //This status code was added since it is deprecated.
         // TODO: remove this after fixing https://github.com/ballerina-platform/ballerina-standard-library/issues/4245
+        httpCodeMap.put("NetworkAuthorizationRequired", "511"); //This status code was added since it is deprecated.
```

Comment:
How is it possible to remove this?

Reply:
HTTP module still have this, may be we can remove it in update 10